src/web_interface/components/miscellaneous_routes.py

```diff
@@ -68,5 +70,39 @@ def _app_delete_firmware(self, uid):
         if not is_firmware:
             return render_template('error.html', message='Firmware not found in database: {}'.format(uid))
         with ConnectTo(AdminDbInterface, config=self._config) as sc:
-            deleted_virtual_file_path_entries, deleted_files = sc.delete_firmware(uid)
-        return render_template('delete_firmware.html', deleted_vps=deleted_virtual_file_path_entries, deleted_files=deleted_files, uid=uid)
+            deleted_virtual_path_entries, deleted_files = sc.delete_firmware(uid)
+        return render_template('delete_firmware.html', deleted_vps=deleted_virtual_path_entries, deleted_files=deleted_files, uid=uid)
+
+    @roles_accepted(*PRIVILEGES['delete'])
+    def _app_find_missing_analyses(self):
+        data = self._find_missing_files()
```

Comment:
`missing_files = ...` ?

Reply:
I wanted to prevent passing the `render_template` function 9999 variables (this time)
renamed (slightly) to `template_data`